schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/besr/SlipTypeEnum.java

```diff
@@ -0,0 +1,50 @@
+package com.jasify.schedule.appengine.besr;
+
+/**
+ * <pre>
+ * 01 = ISR in CHF
+ * 03 = COD-ISR in CHF (cash-on-delivery)
+ * 04 = ISR+ in CHF
+ * 11 = ISR in CHF for credit to own account (section 4.3.3.3)
+ * 14 = ISR+ in CHF for credit to own account (section 4.3.3.3)
+ * 21 = ISR in EUR
+ * 23 = ISR in EUR for credit to own account (section 4.3.3.3)
+ * 31 = ISR+ in EUR
+ * 33 = ISR+ in EUR for credit to own account (section 4.3.3.3)
+ * </pre>
+ *
+ * @author krico
+ * @since 22/07/15.
+ */
+public enum SlipTypeEnum {
+    ISR_CHF("01", "CHF", true),
```

Comment:
Maybe we should use java.util.Currency.
We could also define the currencies we support